build.gradle

```diff
@@ -24,32 +19,32 @@ allprojects {
 }
 
 ext {
-  minSdkVersion = 14
+  minSdkVersion = 21
   targetSdkVersion = 27
-  compileSdkVersion = 27
+  compileSdkVersion = 28
   sourceCompatibilityVersion = JavaVersion.VERSION_1_8
   targetCompatibilityVersion = JavaVersion.VERSION_1_8
 }
 
-def supportLibVersion = '27.1.1'
+def supportLibVersion = '1.1.0'
```

Comment:
unlike before, support libs all have different versions and release schedules. might be worth not declaring this as variable.